packages/nest/src/schematics/library/library.ts

```diff
@@ -0,0 +1,181 @@
+import { normalize, Path } from '@angular-devkit/core';
+import {
+  apply,
+  chain,
+  externalSchematic,
+  filter,
+  MergeStrategy,
+  mergeWith,
+  move,
+  noop,
+  Rule,
+  SchematicContext,
+  template,
+  Tree,
+  url
+} from '@angular-devkit/schematics';
+import {
+  formatFiles,
+  getProjectConfig,
+  names,
+  offsetFromRoot,
+  toFileName,
+  updateJsonInTree,
+  updateWorkspaceInTree,
+  getNpmScope,
+  insert,
+  addGlobal
+} from '@nrwl/workspace';
+import { Schema } from './schema';
+import * as path from 'path';
+import * as ts from '@schematics/angular/third_party/github.com/Microsoft/TypeScript/lib/typescript';
```

Comment:
This is what's causing the `check-imports.js` to fail. 
```suggestion
import * as ts from 'typescript';
```

Reply:
Oh odd ok, webstorm really butchered that import. Thanks